diff --git a/drivers/sensor/mpu6050/mpu6050.c b/drivers/sensor/mpu6050/mpu6050.c
index 34ce2d7d26..baac6f3793 100644
--- a/drivers/sensor/mpu6050/mpu6050.c
+++ b/drivers/sensor/mpu6050/mpu6050.c
@@ -150,6 +150,7 @@ int mpu6050_init(const struct device *dev)
 	const struct mpu6050_config *cfg = dev->config;
 	uint8_t id, i;
 
+	k_msleep(1000);
 	if (!device_is_ready(cfg->i2c.bus)) {
 		LOG_ERR("Bus device is not ready");
 		return -ENODEV;
